source/utilities.cc

```diff
@@ -50,6 +50,46 @@ namespace aspect
     using namespace dealii;
 
     /**
+     * Given an array @p values, consider three cases:
```

Comment:
No need to repeat the documentation a second time here -- it is already fine in the .h file. If you have two copies of two things, they will quickly get out of synch.